src/Lesson11/Task5.java

```diff
@@ -0,0 +1,30 @@
+package Lesson11;
+
+public class Task5 {
+    String text;
+
+    public void isPalindrom(String text) {
+        this.text = text;
+        int k=0;
+
+        for (int i = 0; text.length() > i; i++) {
+            if (text.length() - 1 - i > i) {
+                k++;
+
+                if (text.charAt(text.length() - 1 - i) != text.charAt(i)) {
+                    System.out.println("не є поліндромом");
```

Comment:
I 💓 but don't understand the Ukrainian language, if u have some problem with English, u can use https://translate.google.com/?hl=uk